src/GPSProcessing.py

```diff
@@ -68,8 +68,11 @@ def utc_to_local(utc_dt, lat, lon):
         else:
             # Display the current time in that time zone
             timezone = pytz.timezone(timezone_str)
-            utcoff = timezone.utcoffset(utc_dt)
-            return utcoff.total_seconds()
+            try:
+                utcoff = timezone.utcoffset(utc_dt)
+                return utcoff.total_seconds()
+            except pytz.exceptions.AmbiguousTimeError:
+                print('pytz.exceptions.AmbiguousTimeError: %s' % utc_dt)
```

Comment:
This is not enough to prevent an error. I think when the exception is raised, the timezone should be computed as follow:
```python
utcoff = timezone.utcoffset(utc_dt, is_dst=True)
```
Can you make some tests?

Reply:
http://pytz.sourceforge.net/

Here I found that the `is_dst` parameter is used during ambiguous periods to resolve that ambiguity (only with DST transitions). Now when the exception is raised, we assume DST is in effect and we adjust it

Reply:
Another implementation is the following one:

```
utcoff = timezone.utcoffset(utc_dt, is_dst=None)
return utcoff.total_seconds()
```

When is_dst=None, it raises exception for non-existent or ambiguous times.

I don't preffer this one because we don't get into account the DST effect